Makefile

```diff
@@ -88,11 +97,11 @@ PLATFORMS ?= linux/amd64,linux/arm,linux/arm64
 .PHONY: docker.build
 docker.build: ## Build a local docker container for InGate
 	docker build \
-		--platform=$(PLATFORMS) \
+		--platform $(PLATFORMS) \
```

Comment:
unrelated, buyt why are there docker.build and docker.buildx? Maybe as a suggestion, enforce that all the builds are already done by buildx, and check if the binary exists.

Reply:
probably can be removed, i think it was an issue on my mac and building. build worked and buildx didnt.